pybuoy/observation/observations.py

```diff
@@ -59,3 +59,6 @@ class MeteorologicalObservations(BaseObservations[MeteorologicalObservation]):
 
 class WaveSummaryObservations(BaseObservations[WaveSummaryObservation]):
     ...
+
+class MeteorologicalPredictions(BaseObservations[MeteorologicalPrediction]):
```

Comment:
Modify __repr__ via `super` method

Reply:
yep - and with variable to set print limit as discussed